tests/integration/wordpress_client.py

```diff
@@ -192,3 +200,72 @@ def upload_media(self, filename: str, content: bytes, mimetype: str = None) -> t
         if media["source_url"] not in image_urls:
             image_urls.append(media["source_url"])
         return image_urls
+
+    def associate_ubuntu_one(self, username, password):
+        openid_setting_url = f"http://{self.host}/wp-admin/users.php?page=your_openids"
```

Comment:
We have a new standard for docstrings, do you think it applies here? https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#docstrings

Reply:
Fixed, thanks!